django_project/certification/models/attendee.py

```diff
@@ -70,7 +70,7 @@ class Attendee(models.Model):
     slug = models.SlugField()
     certifying_organisation = \
         models.ForeignKey(CertifyingOrganisation, null=True)
-    author = models.ForeignKey(User)
+    author = models.ForeignKey(User, null=True)
```

Comment:
Why are we allowing the author to be empty?

Reply:
Yes I also made a correction to that, got rid of empty.

Reply:
The changes are in my local as I wanted to fix this migration issue before pushing them.